src/amplitude-client.js

```diff
@@ -225,6 +227,34 @@ AmplitudeClient.prototype.init = function init(apiKey, opt_userId, opt_config, o
   }
 };
 
+/**
+ * @private
+ */
+AmplitudeClient.prototype._migrateUnsentEvents = function _migrateUnsentEvents(cb) {
+  Promise.all([
+      AsyncStorage.getItem(this.options.unsentKey),
+      AsyncStorage.getItem(this.options.unsentIdentifyKey),
+  ]).then((values) => {
+    if (this.options.saveEvents) {
+      var unsentEventsString = values[0];
+      var unsentIdentifyKey = values[1];
+      Promise.all([
+        AsyncStorage.setItem(this.options.unsentKey + this._storageSuffix, unsentEventsString),
+        AsyncStorage.setItem(this.options.unsentIdentifyKey + this._storageSuffix, unsentIdentifyKey ),
```

Comment:
extra space after `unsentIdentifyKey`
side note: would be useful to add a linter or prettier to the JS SDK?